spec/migrations/migration_logs_to_elk_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
```

Comment:
`_to_elk` is not tested here, thus I would rename the file to `migration_logs_spec.rb` as it tests logs written during a migration.